app/lib/backend/model_extensions/backend_api_config.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+module Backend
+  module ModelExtensions
+    module BackendApiConfig
+      extend ActiveSupport::Concern
+
+      included do
+        after_commit :sync_backend_api_metrics_with_backend, on: :create
+      end
+
+      def sync_backend_api_metrics_with_backend
+        backend_api.metrics.each { |metric| metric.send(:sync_backend_for_service, service) }
```

Comment:
this `.send` is already a common thing so imo at this point we should already make those methods public and do `.public_send`